### ISSUE TITLE
Important hyper-parameters to Blenderbot1

### ISSUE THREAD

**Issue by hannnnnxd** [facebookresearch/ParlAI#4001]
Hi,

I am trying to train a 2.7B Blenderbot 1 model in Chinese. I have had a pretrained BART-large model from Fairseq and converted it to ParlAI agent. We built our own datasets according to the project description. Using the default hyper-paramteres of the [BB1 project](https://parl.ai/projects/recipes/), we could achieve a ppl around 11 (the English version is 9.0). Could we get any hint on how to adjust the hyper-parameters to train the BB1 model? e.g., which hyper-paramters are sensetive?

Any other suggestions would be much appreciated!

Best

**Comment by stephenroller** [comment]
Generally learning rate is the thing you want to tune, especially in fine-tuning. I've gotten some small boosts from other things like using `adamw` optimizer, tweaking dropout, etc. But learning rate gets you 90% of the way there.

Of course, PPL's are not comparable across different dictionaries, so I wouldn't expect a Chinese model to have a comparable PPL. A ppl of 11 is pretty good though.
